app/kubemci/pkg/kubeutils/utils.go

```diff
@@ -216,7 +147,18 @@ func (t *KubernetesInformer) getHTTPProbe(svc api_v1.Service, targetPort intstr.
 				}
 			}
 		}
+
 		fmt.Printf("%v: lacks a matching HTTP probe for use in health checks.\n", logStr)
 	}
+
 	return nil, nil
 }
+
+// isSimpleHTTPProbe returns true if the given Probe is:
+// - an HTTPGet probe, as opposed to a tcp or exec probe
+// - has no special host or headers fields, except for possibly an HTTP Host header
+func isSimpleHTTPProbe(probe *api_v1.Probe) bool {
```

Comment:
This is copied from ingress-gce?
Add a comment saying so and add a TODO to refactor to share rather than duplicate.

cc @bowei